src/main/java/nl/tudelft/jpacman/npc/ghost/DispersionMode.java

```diff
@@ -0,0 +1,45 @@
+package nl.tudelft.jpacman.npc.ghost;
+
+import java.util.List;
+
+import nl.tudelft.jpacman.board.Direction;
+
+public class DispersionMode implements Strategy{
+
+	Ghost g;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/mathieu-craeye/jpacman-framework_2/file/2532197015/issues/source?bid=3267846&fileBranchId=3284210#l9)